wikivoyage-listings.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-java -Xmx300m -cp "wikivoyage-listings.jar:build/libs/wikivoyage-listings.jar:lib/*:config/" org.wikivoyage.listings.Main "$@"
+java -Xmx1024m -cp "wikivoyage-listings.jar:build/libs/wikivoyage-listings.jar:lib/*:config/" org.wikivoyage.listings.Main "$@"
```

Comment:
Not sure about that. 

The java process terminates if I set value more than 300m on my cheap VM with 512 MB of RAM. 300m was actually enough for all listings some time ago.

But if speeds up parsing on a box with >2GB of memory, then let's leave 1024m here.